.idea/php.xml

```diff
@@ -37,12 +37,16 @@
       <path value="$PROJECT_DIR$/vendor/symfony/event-dispatcher" />
       <path value="$PROJECT_DIR$/vendor/symfony/debug" />
       <path value="$PROJECT_DIR$/vendor/symfony/console" />
+      <path value="$PROJECT_DIR$/vendor/dhii/psr-14" />
+      <path value="$PROJECT_DIR$/vendor/dhii/stringable-interface" />
+      <path value="$PROJECT_DIR$/vendor/dhii/exception-interface" />
+      <path value="$PROJECT_DIR$/vendor/dhii/factory-interface" />
     </include_path>
   </component>
   <component name="PhpProjectSharedConfiguration" php_language_level="5.3.0" />
   <component name="PhpUnit">
     <phpunit_settings>
-      <PhpUnitSettings load_method="CUSTOM_LOADER" configuration_file_path="F:/projects/dhii/psr-14/phpunit.xml" custom_loader_path="F:/projects/dhii/psr-14/vendor/autoload.php" phpunit_phar_path="" use_configuration_file="true" />
+      <PhpUnitSettings load_method="CUSTOM_LOADER" custom_loader_path="F:/projects/dhii/event-interface/vendor/autoload.php" phpunit_phar_path="" />

```

Comment:
Why is it that this config entry never uses the `$PROJECT_DIR$` variable for you?

Reply:
Mystery. Does it for you?

Reply:
Yes. In fact when I clone projects that have paths like this, after changing the beginning of the path from `F:\projects` to me `D:\dev\` it automatically changes to `$PROJECT_DIR$`.

Reply:
I would love my configs to do that. But they don't. Do you remember what it was like before the PHPS update?

Reply:
No. Would have to go look at some projects that I had created at the time.